### ISSUE TITLE
How to solve the issue about drop of loss of discriminator is too slow ? 

### ISSUE THREAD

**Issue by wangxiao5791509** [imatge-upc/saliency-salgan-2017#14]
Dear authors:
    The following screen shot is my training log, I think the decreasing of loss of discriminator is slow, how can I speed it up ? I first training the network using bce loss, then load pretrained model and continue the training with adversal loss. Do I made any mistakes ? Looking forward for your replay. Thanks very much !!! 

![screenshot from 2017-03-31 22 17 26](https://cloud.githubusercontent.com/assets/12805786/24554640/e620d460-1660-11e7-8680-a432b5a457ba.png)

**Comment by junting** [comment]
Hello @wangxiao5791509 ,
You can probably change the weighting hyperparameter alpha for a smaller value. But it is normal that the drop of adversarial cost is slow to maintain the stability of the adversarial training. 
`class ModelSALGAN(Model):
    def __init__(self, w, h, batch_size=32, G_lr=3e-4, D_lr=3e-4, alpha=1/20.):`
Best,
Junting

**Comment by wangxiao5791509** [comment]
@junting  Thanks for your suggestions.
